src/main/java/seedu/address/model/person/predicates/NameContainsPredicate.java

```diff
@@ -0,0 +1,42 @@
+package seedu.address.model.person.predicates;
+
+import java.util.List;
+
+import seedu.address.commons.util.ToStringBuilder;
+import seedu.address.model.person.Person;
+
+/**
+ * Tests that a {@code Person}'s {@code Name} matches any of the keywords given.
+ */
+public class NameContainsPredicate extends SupplierPredicate {
```

Comment:
Could this class be named better? Perhaps it can be called SupplierContainsNamePredicate?

Reply:
It was meant to be named NameContainsKeywordPredicate